src/main/kotlin/no/nav/syfo/api/ApiModule.kt

```diff
@@ -15,7 +14,7 @@ import no.nav.syfo.personoppgavehendelse.PersonoppgavehendelseProducer
 
 fun Application.apiModule(
     applicationState: ApplicationState,
-    azureAdV2Client: AzureAdV2Client,
```

Comment:
Denne var ikke i bruk, eller?

Reply:
Nei, den brukes nå et hakk lenger ut her for å instansiere opp VeilederTilgangskontrollClient.